### PR TITLE
js apiview parser build pipeline

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Create a pipeline to build and release JS APIView parser to devops feed

Fixes: #3923

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
Verified publishing APIView parser for JS to devops feed.
Job: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1774469&view=logs&j=826b5e63-3908-5d05-b7a1-4a06c78abfa8&t=c1e5fd86-9d32-5c34-9fa3-6050485eac25

**Comment by praveenkuttappan** [comment]
@weshaggard Please let me know your feedback about this PR to publish JS parser

File: src/ts/ts-genapi/ci.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```
Comment (weshaggard) [inline src/ts/ts-genapi/ci.yml:1]: Can we put these files under tools and/or apiview? As it stands it isn't clear that this is used for ApiView.

Comment (weshaggard) [reply src/ts/ts-genapi/ci.yml:1]: Also can probably improve the "ts-genapi" name a little to make it clearer what it is for.

File: src/ts/ts-genapi/ci.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        displayName: 'Publish ts-genapi package to devops feed'
+        pool:
+          name: azsdk-pool-mms-ubuntu-2004-general
+          vmImage: MMSUbuntu20.04
+        steps:
+        - checkout: none
+        - download: current
+        
+        - pwsh: |
+            $detectedPackageName=Get-ChildItem $(Pipeline.Workspace)/$(ArtifactName)/*.tgz
```
[the remark this answers is not in the document]
Comment (praveenkuttappan) [reply src/ts/ts-genapi/ci.yml:84]: Nope. It's in the JS repo. I can submit a separate PR to make it common script.

[the remark this answers is not in the document]
Comment (weshaggard) [reply src/ts/ts-genapi/ci.yml:84]: We have https://github.com/Azure/azure-sdk-tools/issues/2153 which is tracking that work.

**Pull request merged**